[esx]/esx_multicharacter/client/main.lua

```diff
@@ -140,6 +140,24 @@ if ESX.GetConfig().Multichar then
 		})
 	end
 
+	function CharacterDeleteConfirmation(Characters, slots, SelectedCharacter, value)
+		local elements = {
+			{title = 'Delete Confirmation', icon = "fa-solid fa-users", description = 'Are you sure removing selected character?', unselectable = true},
+			{title = TranslateCap('char_delete'), icon ="fa-solid fa-xmark", description = 'Yes, I am sure removing selected character', action = 'delete', value = value},
+			{title = TranslateCap('return'), unselectable = false, icon = "fa-solid fa-arrow-left", description = 'No, return to character options', action = "return"}
```

Comment:
please add translate for all texts

Reply:
I added translation on English only, is that fine?